source/modules/collection-api/transfer/overview.md.erb

```diff
@@ -1,19 +1,16 @@
-## Items Transfers
+## Item Transfers
 
-The following functions will be used to relay item (*e.g. ticket*) transfers between users (*e.g. accounts*) that happen within your platform:
+You can provide transfer-related activity via the REST API. The `/transfer` endpoint supports POST and PUT operations with the following required arguments:
 
-- **Item Transfer Creation** (*trackItemTransferCreation*)
-- <a href="#transfer-creation">trackTransferCreation</a>
-- <a href="#transfer-updates">trackTransferUpdate</a>
-- **Item Transfer Update** (*trackItemTransferUpdate*)
-
-All the functions for tracking transfer activity require a [InspetorTransfer](#inspetortransfer) object as argument and return `true` if everything goes right. Otherwise, one of the the following exceptions will be thrown:
-
-Exception | Description
---------- | -----------
-**[TransferException](#inspetortransferexception)**  | The Transfer object provided is not a valid one
-**[generalException](#inspetorgeneralexception)**  | The timestamps that you passed on the objects are not valid ones
-**[TrackerException](#trackerexception)**    | An internal error occurred *Hopefully this never happen*
+Property                 | When is it required? | Type    | Description
+--------                 | -------------------- | ----    | -----------
+id                       | `POST`, `UPDATE`     | String  | Your platform-unique identifier for the transfer
+sale_id                  | `POST`               | String  | Your platform-unique identifier for the **sale** associated with the items being transfered
+sale_item_id             | `POST`               | String  | The id of the [sale item](#sale-items) (*e.g. ticket*) that is being transfered
+sender_account_id        | `POST`               | String  | The id of the account who is transferring the item (*e.g. ticket*)
+recipient_email          | `POST`               | String  | The email of the user who is receiving the item
+recipient_account_id     |                      | String  | The id of the account (if available) who is receiving the item
+status                   | `POST`, `UPDATE`     | String  | The status of the transfer. *The allowed values are: `accepted`, `rejected`, and `pending`*
```

Comment:
Maybe change `accepted` to `approved` and `rejected` to `declined`, just so we follow the same pattern as in sales?